docs/how-to-run-the-solution-in-azure/README.md

```diff
@@ -16,19 +16,27 @@ This document describes the resources that must be created and configured to run
         - [Application Insights](application_insights.md).
         - [Storage account](storage_account.md).
         - [Cosmos DB](cosmos_db.md).
-        - [Web App and App service plan](web_app_and_app_service_plan.md).
-        - [Function App and App service plan](function_app_and_app_service_plan.md).
+        - [App Service Plan](service_plan.md).
+        - [Web App](web_app_and_app_service_plan.md).
+        - [Function App](function_app_and_app_service_plan.md).
     - Virtual Machine resource group:
-        - [Virtual Machine](bot_service_virtual_machine.md).
+        - [Virtual Machine](virtual_machine.md).
         - [Event Grid](configure_event_grid.md).
 - Components Deployment:
+    - [Deploy the Bot Service into the Virtual Machine](deploy_bot-service.md).
```

Comment:
**nitpick (blocking)**: Fix this broken link. Replace _deploy_bot-service.md_ with _deploy_bot_service.md_